pkg/controller/windowsmachine/nodeconfig/nodeconfig.go

```diff
@@ -136,6 +136,12 @@ func (nc *nodeConfig) Configure() error {
 	if err := nc.configureNetwork(); err != nil {
 		return errors.Wrap(err, "configuring node network failed")
 	}
+	// Start Windows metrics exporter service, only if the file is present on the VM
+	if _, err1 := nc.Windows.Run("Test-Path "+windows.WindowsExporterPath, true); err1 == nil {
```

Comment:
can we use the windowsExporterPath from the wellknownlocations here?

Reply:
I think we can't as the wkl path refers to that on the wmco image rather than the file's location on the Windows VM itself.